modules/migration-about-data-copy-methods.adoc

```diff
@@ -1,10 +1,10 @@
 // Module included in the following assemblies:
 //
-// migration/migrating_3_4/configuring-replication-repository-3-4.adoc
-// migration/migrating_4_1_4/configuring-replication-repository-4-1-4.adoc
-// migration/migrating_4_2_4/configuring-replication-repository-4-2-4.adoc
-[id='migration-understanding-data-copy-methods_{context}']
-= Understanding the data copy methods for migration
+// * migration/migrating_3_4/migrating-application-workloads-3-4.adoc
+// * migration/migrating_4_1_4/migrating-application-workloads-4-1-4.adoc
+// * migration/migrating_4_2_4/migrating-application-workloads-4-2-4.adoc
+[id='migration-about-data-copy-methods_{context}']
```

Comment:
Same in this file about reverting the file rename and anchor, and fixing the assembly list.